eng/runTestsCannotRunOnHelix.ps1

```diff
@@ -1,3 +1,6 @@
+#hack to make sure build of helix run is done, so no file lock
+Start-Sleep -Seconds 90
```

Comment:
I'd love to see us eventually get rid of this hack.

Reply:
Agree, we should eventually move all tests to helix. And we can remove that part completely.